bin/node/runtime/src/weights/pallet_vesting.rs

```diff
@@ -0,0 +1,63 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0-rc6
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::weights::{Weight, constants::RocksDbWeight as DbWeight};
+
+pub struct WeightInfo;
+impl pallet_vesting::WeightInfo for WeightInfo {
```

Comment:
as nudged to myself as well, need to be implemented for `()`, or did we re-consider? :p

Reply:
ahh this file is probably need to be removed.

Reply:
What do you mean here?

I implement `()` in the `default_weight.rs` file, but I also define it here because it illustrates how weights should be used for individual runtimes. A runtime should not depend on `()` unless it is a test runtime since it can change from upstream.

Reply:
yes, my misunderstanding.